examples/managers/my_new_etl.py

```diff
@@ -330,6 +330,7 @@ def name(cls) -> str:
     def relative_path(self) -> pathlib.Path:
         return super().relative_path() / "min"
 
+    @classmethod
     def data_var(self) -> str:
```

Comment:
For class methods by convention use `cls` instead of `self` just because the first arg will be a class not an instance. We might also want to adopt `abstract_class_property` from https://github.com/Arbol-Project/gridded-etl-tools/pull/91